src/settings.c

```diff
@@ -106,6 +106,12 @@ void load_settings(char *cmdline_config_path)
               "Using STATIC_CONFIG is deprecated behavior.");
 #endif
 
+        log_set_level_from_string(option_get_string(
```

Comment:
Also there's probably a memory leak here, the return of `option_get_string` is never freed.

Reply:
This is a definitive memory leak. It's just that I actually want to change change the settings parser in #475 to return `const` values. This makes in such a case much more sense.

I had in mind, that this will be a memory leak, but only temporary. I've actually thought this will get fixed up in a follow up commit in this PR. But obviously this commit is on #475.

I guess, I'll fix it up to later refix it.